content/gotraining.slide

```diff
@@ -339,6 +339,65 @@ Don't be afraid!
 Try it out on a new, throwaway branch.
 
 
+* Rebase interactively
+
+[[https://stackoverflow.com/a/36644561][Follow instruction]] to setup VSCode as `git` editor (iff that's your preferred editor).
```

Comment:
s/iff/if/ - i assume you don't mean iff.

Reply:
I did mean `if and only if`

Reply:
I did mean `if and only if`

Reply:
But now that I've read up on what it really means it makes no sense at all... I think.

Reply:
But now that I've read up on what it really means it makes no sense at all... I think.